source/04-MX/02.ptx

```diff
@@ -90,105 +90,110 @@ and a group that means <m>T</m> is <em>bijective</em>.
-<term>inverse matrix</term><idx>inverse matrix</idx> of <m>A</m>, so we also say that <m>A</m> is <term>invertible</term><idx>invertible</idx>.
-</p></li>
-</ul>
     </statement>
 </definition>
 
 
 <activity estimated-time='20'>
     <introduction>
         <p>
-  Let <m>T: \IR^3 \rightarrow \IR^3</m> be the linear transformation given by the standard matrix
-  <m>A=\left[\begin{array}{ccc} 2 &amp; -1 &amp; -6 \\ 2 &amp; 1 &amp; 3 \\ 1 &amp; 1 &amp; 4 \end{array}\right]</m>.
+Let <m>T: \IR^3 \rightarrow \IR^3</m> be the linear bijection given by the standard matrix
+<m>A=\left[\begin{array}{ccc} 2 &amp; -1 &amp; -6 \\ 2 &amp; 1 &amp; 3 \\ 1 &amp; 1 &amp; 4 \end{array}\right]</m>.
         </p>
     </introduction>
   <task>
       <p>
-  Write an augmented matrix representing the system of equations given by <m>T(\vec x)=\vec{e}_1</m>,
-  that is, <m>A\vec x=\left[\begin{array}{c}1 \\ 0 \\ 0 \end{array}\right]</m>.
-  Then solve <m>T(\vec x)=\vec{e}_1</m> to find <m>T^{-1}(\vec{e}_1)</m>.
+To find <m>\vec x = T^{-1}(\vec{e}_1)</m>, we need to find the unique
+solution for <m>T(\vec x)=\vec e_1</m>. Which of these linear systems
+can be used to find this solution?
+        <ol marker="A." cols="2">
+            <li><m>
+\begin{array}{cccc}
+2x_1 &amp; -1x_2 &amp; -6x_3 &amp; =x_1 \\ 
+2x_1 &amp; +1x_2 &amp; +3x_3 &amp; =0 \\ 
+1x_1 &amp; +1x_2 &amp; +4x_3 &amp; =0
+\end{array}
+            </m></li>
+            <li><m>
+\begin{array}{cccc}
+2x_1 &amp; -1x_2 &amp; -6x_3 &amp; =x_1 \\ 
+2x_1 &amp; +1x_2 &amp; +3x_3 &amp; =x_2 \\ 
+1x_1 &amp; +1x_2 &amp; +4x_3 &amp; =x_3
+\end{array}
+            </m></li>
+            <li><m>
+\begin{array}{cccc}
+2x_1 &amp; -1x_2 &amp; -6x_3 &amp; =1 \\ 
+2x_1 &amp; +1x_2 &amp; +3x_3 &amp; =0 \\ 
+1x_1 &amp; +1x_2 &amp; +4x_3 &amp; =0
+\end{array}
+            </m></li>
+            <li><m>
+\begin{array}{cccc}
+2x_1 &amp; -1x_2 &amp; -6x_3 &amp; =1 \\ 
+2x_1 &amp; +1x_2 &amp; +3x_3 &amp; =1 \\ 
+1x_1 &amp; +1x_2 &amp; +4x_3 &amp; =1
+\end{array}
+            </m></li>
+        </ol>
       </p>
   </task>
   <task>
       <p>
-   Solve <m>T(\vec x)=\vec{e}_2</m> to find <m>T^{-1}(\vec{e}_2)</m>.
+Use that system to find the solution <m>\vec x=T^{-1}(\vec{e}_1)</m>
+for <m>T(\vec x)=\vec{e}_1</m>.
       </p>
   </task>
   <task>
       <p>
-   Solve <m>T(\vec x)=\vec{e}_3</m> to find <m>T^{-1}(\vec{e}_3)</m>.
+Similarly, solve <m>T(\vec x)=\vec{e}_2</m> to find <m>T^{-1}(\vec{e}_2)</m>,
+and solve <m>T(\vec x)=\vec{e}_3</m> to find <m>T^{-1}(\vec{e}_3)</m>.
       </p>
-  </task>
-  <task>
+    </task>
+    <task>
       <p>
-   Write <m>A^{-1}</m>, the standard matrix for <m>T^{-1}</m>.
+Use these to write <me>
+A^{-1}= [T^{-1}(\vec e_1)\hspace{1em}
+T^{-1}(\vec e_2)\hspace{1em}T^{-1}(\vec e_3)]
+</me>, the standard matrix for <m>T^{-1}</m>.
       </p>
   </task>
 </activity>
 
 <sage language="octave">
 </sage>
 
-<observation>
```

Comment:
I removed this observation as I'd prefer students to take extra time to break down into the inverse transformations of standard basic vectors. (A quick RREF shortcut isn't needed when the inverse can just be computed directly anyway.)